scripts/projection_script.r

```diff
@@ -0,0 +1,65 @@
+setwd("/home/tec0/covid_booster_uptake")
+source("get_uptake_data.r")
+source("interpolate_dates.r")
+source("build_scaling_model.r")
+source("evaluate_scaling.r")
+source("build_projection_model.r")
+source("evaluate_projection.r")
+source("projection_functions.r")
+
```

Comment:
You can also use `devtools::load_all(the_directory_with_all_source_files)` or `load_code()`